asv_bench/benchmarks/series_methods.py

```diff
@@ -166,6 +175,19 @@ def time_value_counts(self, N, dtype):
         self.s.value_counts()
 
 
+class ValueCountsEa:
```

Comment:
```suggestion
class ValueCountsEA:
```

? Small nitpick: not sure if we have some consistency around this, but if not, since we typically use EA as abbreviation, would keep it capitalized here

Reply:
Done, will keep in mind for future pro